news/2020-06-21-project-loved-june-2020.md

```diff
@@ -0,0 +1,210 @@
+
+The chart features much difference in approach from the traditional OtFS, with a greater yet subtler focus on musical detail and intricacy. Its cleanliness lends itself nicely to player practice, with its different rates providing adequate coverage as challenge for the many keen-eyed players out there.
+
+Contrary to its name, You Can (sometimes) Redo, and it's clear Shoegazer's call has sufficed.
+
+[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/mania/2-gympie-gympe.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[QUIL - Gympie Gympe](https://osu.ppy.sh/beatmapsets/560952#mania) by [Valedict](https://osu.ppy.sh/users/2537082)  
+242 BPM, 2:23 | \[4K\], 4.98★ – 6.50★
+
+Repetitive musical motifs and the subsequent deliberate usage of repetitive patterns seems to be a trend for the first two charts in osu!mania's Loved nominations, with the second being Valedict's *Gympie Gympe*! The chart is a relatively slow jack chart, with the jacks clocking in at 115 BPM, that uses the repeating and developing sounds to push the jacks' density further and further as the chart progresses. This makes playing through not only satisfying, but also allows for the intensity of the play to increase as much as possible until the very end of the chart, making it extremely engaging no matter the skill level of the player. Not only this, but the mapset comes with many extra difficulties with higher rates, making it a great practice chart for people looking to improve their jacks, and a great challenge chart for the top-end players to show off their skills. Ultimately, *Gympie Gympe* became very popular as a result of these things on osu!mania and StepMania/Etterna, and has dug itself a fitting spot among the Loved nominations.
+
+[![](/wiki/shared/news/2020-06-21-project-loved-june-2020/mania/3-warriors.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Hommarju - Warriors](https://osu.ppy.sh/beatmapsets/376832#mania) by [ALEFY](https://osu.ppy.sh/users/1291769)  
+128 BPM, 2:01 | \[4K\] 3.29★, \[4K\] 3.29★
+
+Up next is an extremely simple nomination, but one that has gained an immense amount of popularity since it's upload. The map is ALEFY's *Warriors*, a staple of his SV maps that heavily utilizes red line gimmicks. The main draw of the map comes in its choruses, where this usage of red lines is most obvious: instantaneous teleports and 0 BPM lines create a seamless, sudden appearance of following notes and make for an interesting experience similar to that in [maskeara](https://osu.ppy.sh/users/2393914)'s Loved set of *[POSSESSION](https://osu.ppy.sh/beatmapsets/551469)* set. It's no wonder that *Warriors* gained the following it has today given these simple, yet very enjoyable SV gimmicks that keep players coming back. This makes it a clear shoo-in for Loved.
```

Comment:
```suggestion
Up next is an extremely simple nomination, but one that has gained an immense amount of popularity since it's upload. The map is ALEFY's *Warriors*, a staple of his SV maps that heavily utilizes red line gimmicks. The main draw of the map comes in its choruses, where this usage of red lines is most obvious: instantaneous teleports and 0 BPM lines create a seamless, sudden appearance of following notes and make for an interesting experience similar to that in [maskeara](https://osu.ppy.sh/users/2393914)'s Loved set of *[POSSESSION](https://osu.ppy.sh/beatmapsets/551469)* set. 

It's no wonder that *Warriors* gained the following it has today given these simple, yet very enjoyable SV gimmicks that keep players coming back. This makes it a clear shoo-in for Loved.
```